lib/flame_on/capture.ex

```diff
@@ -2,24 +2,46 @@ defmodule FlameOn.Capture do
   @output_directory "flame_on_output_delete_me"
   @check_interval 500
 
-  def capture(module, function, arity, pid, id, timeout \\ 15_000) do
-    File.rm_rf!(@output_directory)
-    File.mkdir!(@output_directory)
+  def capture(module, function, arity, opts) do
+    target_node = Keyword.get(opts, :target_node, Node.self())
 
-    :eflambe.capture({module, function, arity}, 1, output_directory: @output_directory)
-    watch_output_directory(pid, id, timeout)
+    Node.spawn(target_node, __MODULE__, :__local_capture__, [
```

Comment:
`:eprc`, or `:rpc` (if you want older OTP's) is what you want here. If you stay with `Node.spawn` approach, you should use use `make_ref()` and match on the ref instead of the private atom. You should also use `spawn_link`, but the rpc functions handle all these details so they would be preferred

Reply:
Updated. As discussed in slack, we need to `Node.spawn` as the process is not ending immediately which rules out an `:erpc`, so I swapped to using refs and spawn_link as suggested.